migration/README.md

```diff
@@ -0,0 +1,47 @@
+# Database Schema Migration
+
+For the database schema migration, we use the [Atlas tool](https://github.com/ariga/atlas).
+
+When you install the Bucketeer Helm application, the migration runs automatically before the Bucketeer application is installed.
+If the migration fails, it won't install the application.
+
+## Prerequisite
+
+Ensure that you have started the Minikube in the dev container and that the `localenv-mysql-0` pod is running without errors.
+
+## 1- Connecting to MySQL
+
+To connect to MySQL, you must enter the `localenv-mysql-0` pod and then connect to MySQL.
+
+```shell
+kubectl exec -it localenv-mysql-0 -- /bin/sh
+mysql -h localhost -u bucketeer -p bucketeer
+```
+
+Once you are logged in, you can make your changes in the Database.
+
+## 2- Creating Migration File
+
+To create the migration file, you must port-forward the `localenv-mysql` service before creating it.
+
+```shell
+kubectl port-forward svc/localenv-mysql 3306:3306
+```
+
+The following command will create the migration file.
+
+```shell
+make create-migration NAME=<MIGRATION_FILE_NAME> HOST=localhost USER=bucketeer PASS=bucketeer PORT=3306 DB=bucketeer
```

Comment:
I got an error;

```sh
make create-migration NAME=create_foo_table HOST=localhost USER=bucketeer PASS=bucketeer PORT=3306 DB=bucketeer
# Example: make create-migration NAME=create_table_users USER=root PASS=password HOST=localhost PORT=3306 DB=bucketeer
atlas migrate diff create_foo_table \
        --dir file://migration/mysql \
        --to mysql://bucketeer:bucketeer@localhost:3306/bucketeer \
        --dev-url docker://mysql/8/bucketeer
make: atlas: No such file or directory
make: *** [Makefile:264: create-migration] Error 127
```

Reply:
If you are running it in the dev container and get the error, you are probably running an old image.
Try to delete the current image locally and pull it again.

https://github.com/bucketeer-io/bucketeer/pkgs/container/bucketeer-devcontainer
https://github.com/bucketeer-io/bucketeer/blob/main/.github/.devcontainer/Dockerfile#L94

Reply:
It worked :+